scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -657,7 +655,6 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
                 {
                     // Yes, it is .exe even on Linux. This is the managed exe we're working with
                     File.Copy(Path.Combine(binaryToCorehostifyOutDir, $"{binaryToCorehostify}.exe"), Path.Combine(outputDir, $"{binaryToCorehostify}.dll"));
-                    File.Delete(Path.Combine(binaryToCorehostifyOutDir, $"{binaryToCorehostify}.exe"));
```

Comment:
I've added renaming code because `csc.exe` could not be ran directly with portable cli setup. Having two csc's does not make sense, but you can change `ChangeEntryPointLibraryName` second argument to null and it will delete deps entry completely.